README.md

```diff
@@ -97,8 +97,9 @@ You can load many more quantized models hosted on the hub under the Intel organi
 Check out the [`examples`](https://github.com/huggingface/optimum-intel/tree/main/examples) directory for more sophisticated usage.
 
 ### OpenVINO
+Below are the examples of how to use OpenVINO and its [NNCF](https://docs.openvino.ai/latest/tmo_introduction.html) tool for model optimization and inference.
```

Comment:
```suggestion
Below are the examples of how to use OpenVINO and its [NNCF](https://docs.openvino.ai/latest/tmo_introduction.html) framework for model optimization, quantization, and inference.
```
NNCF framework is doubling framework, but to me tool does not cover it.
IMO it would be good to mention quantization specifically, to prevent confusion about the kind of optimization OpenVINO Runtime does and what NNCF does.